Source/Extensions/Blazorise.RichTextEdit/RichTextEditToolbarGroup.razor.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Components;
+
+namespace Blazorise.RichTextEdit
+{
+    public partial class RichTextEditToolbarGroup
+    {
+        #region Members        
+        /// <summary>The class provider.</summary>
+        [Inject] private IClassProvider classProvider { get; set; }
+
+        /// <summary>The constructed class names</summary>
+        private string classNames;
```

Comment:
Use ClassBuilder for class names.

Reply:
Changed this component to inherit `BaseComponent` and use that logic